install/install_db.php

```diff
@@ -0,0 +1,6621 @@
+      ->addUniqueKey('unicity', ['ext'])
+      ->createOrDie(true);
+
+   $table_schema->init('glpi_domains')
+      ->addIndexedField('name', 'string')
+      ->addField('entities_id', 'int', ['value' => '0'])
+      ->addField('is_recursive', 'boolean', ['value' => '0'])
+      ->addField('comment', 'text')
+      ->addIndexedField('date_mod', 'timestamp')
+      ->addIndexedField('date_creation', 'timestamp')
+      ->createOrDie(true);
+
+   $table_schema->init('glpi_dropdowntranslations')
+      ->addField('items_id', 'int', ['value' => '0'])
+      ->addField('itemtype', 'varchar(100)')
+      ->addIndexedField('language', 'varchar(5)')
+      ->addIndexedField('field', 'varchar(100)')
+      ->addField('value', 'text')
+      ->addUniqueKey('unicity', ['itemtype', 'items_id', 'language', 'field'])
+      ->addKey('typeid', ['itemtype', 'items_id'])
+      ->createOrDie(true);
+
+   $table_schema->init('glpi_enclosuremodels')
+      ->addIndexedField('name', 'string')
+      ->addField('comment', 'text')
+      ->addIndexedField('product_number', 'string')
+      ->addField('weight', 'int', ['value' => '0'])
+      ->addField('required_units', 'int', ['value' => '1'])
+      ->addField('depth', 'float', ['value' => '1'])
+      ->addField('power_connections', 'int', ['value' => '0'])
+      ->addField('power_consumption', 'int', ['value' => '0'])
+      ->addField('is_half_rack', 'boolean', ['value' => '0'])
+      ->addField('picture_front', 'text')
+      ->addField('picture_rear', 'text')
+      ->addIndexedField('date_mod', 'timestamp')
+      ->addIndexedField('date_creation', 'timestamp')
+      ->createOrDie(true);
+
+   $table_schema->init('glpi_enclosures')
+      ->addField('name', 'string')
+      ->addIndexedField('entities_id', 'int', ['value' => '0'])
+      ->addIndexedField('is_recursive', 'boolean', ['value' => '0'])
+      ->addIndexedField('locations_id', 'int', ['value' => '0'])
+      ->addField('serial', 'string')
+      ->addField('otherserial', 'string')
+      ->addIndexedField('enclosuremodels_id', 'int')
+      ->addIndexedField('users_id_tech', 'int', ['value' => '0'])
+      ->addField('groups_id_tech', 'int', ['value' => '0'])
+      ->addIndexedField('is_template', 'boolean', ['value' => '0'])
+      ->addField('template_name', 'string')
+      ->addIndexedField('is_deleted', 'boolean', ['value' => '0'])
+      ->addField('orientation', 'boolean')
+      ->addField('power_supplies', 'boolean', ['value' => '0'])
+      ->addIndexedField('states_id', 'int', ['value' => '0', 'nodefault' => false, 'comment' => 'RELATION to states (id)'])
+      ->addField('comment', 'text')
+      ->addIndexedField('manufacturers_id', 'int', ['value' => '0'])
+      ->addField('date_mod', 'timestamp')
+      ->addField('date_creation', 'timestamp')
+      ->addKey('group_id_tech', ['groups_id_tech'])
+      ->createOrDie(true);
+
+   $table_schema->init('glpi_entities', false)
```

Comment:
I'm not sure why, but glpi_entities did not have AUTO INCREMENT on the id field.